recipe/meta.yaml

```diff
@@ -12,7 +12,7 @@ requirements:
   run:
     - dask =2023.9.2
     - distributed =2023.9.2
-    - dask-gateway =2023.1.1
+    - dask-gateway =2023.9.0
```

Comment:
we may want / need to hold on merging this until the dask helm chart makes an equivalent release.

Reply:
https://helm.dask.org has

dask-gateway-2023.9.0 | 2023.9.0 | 26 September 2023
-- | -- | --

so i'll go ahead and merge this